src/AccessibilityInsights.Extensions.AzureDevOps/IssueTemplates/IssueColorContrast.html

```diff
@@ -9,15 +9,15 @@
 <b>To Reproduce and How to Fix guidance:</b> Please open the attached Accessibility Insights for Windows Test File (*.a11ytest) and use the color picker dropdown to find a color that meets the necessary color contrast ratio.
 <br/>
 <br/>
-This accessibility issue was found using <b>Accessibility Insights for Windows</b>, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at <a href="https://aka.ms/AccessibilityInsights">https://aka.ms/AccessibilityInsights</a>.<br/>
+This accessibility issue was found using <b>Accessibility Insights for Windows</b>, a tool that helps dedebug and find accessibility issues earlier. Get more information & download this tool at <a href="https://aka.ms/AccessibilityInsights">https://aka.ms/AccessibilityInsights</a>.<br/>
```

Comment:
typo - dedebug - double negation so a bug? :-P

Reply:
That's one way of looking at it!